pokete_data/map_data.py

```diff
@@ -1928,7 +1928,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt": """(())(()()))(())()()()))((())())(()))()())()\___   \________       \______            \        \\
+                "txt": """
```

Comment:
This also produces a CoordinateError.

Reply:
Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08